dataland-frontend/tests/component/components/general/ActivityFormField.cy.ts

```diff
@@ -13,7 +13,7 @@ describe('Component test for ActivitiesFormField', () => {
     cy.get('ul.p-treenode-children')
       .should('be.visible')
       .find('li.p-treenode')
-      .should('have.length', 7)
+      .should('have.length', 10)
       .eq(1)
```

Comment:
Why is this needed?

Reply:
The test checks how many items there are clustered in this specific Activities form field (I think it was "Construction something"). Since there were new Activities added to this cluster, the length increased.